AsyncDisplayKit/ASTextNode.mm

```diff
@@ -351,9 +351,8 @@ - (CGSize)calculateSizeThatFits:(CGSize)constrainedSize
   
   CGSize size = [self _renderer].size;
   if (_attributedText.length > 0) {
-    CGFloat screenScale = ASScreenScale();
-    self.ascender = round([[_attributedText attribute:NSFontAttributeName atIndex:0 effectiveRange:NULL] ascender] * screenScale)/screenScale;
-    self.descender = round([[_attributedText attribute:NSFontAttributeName atIndex:_attributedText.length - 1 effectiveRange:NULL] descender] * screenScale)/screenScale;
+    self.ascender = [[self class] ascenderWithAttributedString:_attributedText];
+    self.descender = [[_attributedText attribute:NSFontAttributeName atIndex:_attributedText.length - 1 effectiveRange:NULL] descender];
```

Comment:
What is the rationale for no longer rounding ascender and descender to the nearest pixel? I believe @rcancro implemented those properties on the layout side, maybe he has some input here?